pkg/services/featuremgmt/openfeature.go

```diff
@@ -17,6 +17,46 @@ const (
 	featuresProviderAudience = "features.grafana.app"
 )
 
+// OpenFeatureConfig holds configuration for initializing OpenFeature
+type OpenFeatureConfig struct {
+	// ProviderType is either "static" or "goff"
+	ProviderType string
+	// URL is the GOFF service URL (required for GOFF provider)
+	URL *url.URL
+	// HTTPClient is a pre-configured HTTP client (optional, used for GOFF provider)
+	HTTPClient *http.Client
+	// StaticFlags are the feature flags to use with static provider
+	StaticFlags map[string]bool
+	// TargetingKey is used for evaluation context
+	TargetingKey string
+	// ContextAttrs are additional attributes for evaluation context
+	ContextAttrs map[string]any
+}
+
+// InitOpenFeature initializes OpenFeature with the provided configuration
+func InitOpenFeature(config OpenFeatureConfig) error {
+	// For GOFF provider, ensure we have a URL
+	if config.ProviderType == setting.GOFFProviderType && (config.URL == nil || config.URL.String() == "") {
+		return fmt.Errorf("URL is required for GOFF provider")
+	}
+
+	// Initialize the provider
+	err := initOpenFeature(config.ProviderType, config.URL, config.StaticFlags, config.HTTPClient)
```

Comment:
nit: having `InitOpenFeature` and the same `initOpenFeature` can be a bit confusing. `initOpenFeature` is only used here so it doesn't need to be a different function?